spec/features/admin_views_users_order_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.feature "Admin Views Users Single Order", type: :feature do
+  # As an authenticated Admin, when I visit an individual order page
+  # Then I can see the order's date and time.
+  # And I can see the purchaser's full name and address.
+  # And I can see, for each item on the order:
+  # - The item's name, which is linked to the item page.
+  # - Quantity in this order.
+  # - Price
+  # - Line item subtotal.
+  # And I can see the total for the order.
+  # And I can see the status for the order.
+  scenario "admin can see order date and time" do
+    create(:order_with_items)
+
+    admin = create(:user, role: 1)
+    login_user(admin)
+    click_link("#1")
+
+    expect(page).to have_content(Time.now.strftime("%m/%d/%Y %I:%M%p"))
+  end
+
+  scenario "admin can see purchaser's full name and address" do
+    create(:order_with_items)
+
+    admin = create(:user, role: 1)
+
+    login_user(admin)
+    click_link("#1")
+
+    expect(page).to have_content("Purchaser Name: Joe Delaware")
+    expect(page).to have_content("Address: 4 Kemper Lane, Salt Lake City, Utah - 84104")
```

Comment:
I'm afraid that the since the address is different n Kemper Lane that this test will fail occasionally. Perhaps `have_content("Kemper Lane, Salt Lake City, Utah - 84104")` instead?